build/DotNetSdkTestVersions.txt

```diff
@@ -1,4 +1,4 @@
 # Each line represents arguments for the .NET SDK installer script (https://learn.microsoft.com/dotnet/core/tools/dotnet-install-script)
```

Comment:
Is it worth adding a comment that the SDK should always come last?

Reply:
It worked for years this way, I'm currently writing to the SDK team to ask them if this is intentional, or just a bug in one of the recently built SDKs.  Depending on the outcome, it may or may not be needed.

Reply:
Ok, we were always doing it wrong and were just lucky that it didn't break earlier.  Comment added.